python/paddle/fluid/dygraph/varbase_patch_methods.py

```diff
@@ -133,6 +133,10 @@ def backward(self, backward_strategy=None):
 
         Args:
             backward_strategy( :ref:`api_fluid_dygraph_BackwardStrategy` ): The Backward Strategy to run backward
+            retain_graph(bool, optional): If False, the graph used to compute grads will be freed. After calling
+            this method(`backward`), if you still want to add more ops to the graph previously built, you have to
```

Comment:
English issue:

"you want", "I want" is not good manners in English. You can change the sentence as:

"If you would like to add more ops to the built graph after calling this method(`backward`), set the ..."

Reply:
Thanks. Done.